scripts/Tools/testreporter.py

```diff
@@ -0,0 +1,203 @@
+#!/usr/bin/env python
```

Comment:
Followed the template for python scripts.  create_clone was a good simple example.